scipy/stats/_kde.py

```diff
@@ -371,7 +364,7 @@ def integrate_box_1d(self, low, high):
                         special.ndtr(normalized_low)))
         return value
 
-    def integrate_box(self, low_bounds, high_bounds, maxpts=None):
+    def integrate_box(self, low_bounds, high_bounds, maxpts=None, **kwds):
```

Comment:
`**kwds` isn't documented in this PR, and this is a public method. Is this actually necessary? It ties `integrate_box` to current and future implementation details of `multivariate_normal`.

Reply:
Yeah, not strictly necessary, no. Currently it is an escape hatch for when (if and when, actually) the "new" integrator is not exactly backwards compatible and then a user can fiddle with `rng` and `abserr` knobs of  `multivariate_normal.cdf`. Can definitely take it out, at least until there's an explicit user request.

Reply:
+1 for taking it out then

Reply:
OK, done.